### PR TITLE
systemd-journal-remote: Remove old files regularly

### PR DESCRIPTION
**Pull request by danrue**
Remove files older than 14 days daily.

Signed-off-by: Dan Rue <dan.rue@linaro.org>

### CODE REVIEW

**Comment by danrue** [comment]
This will get deployed during the next squad release; it looks a little difficult to deploy now safely.

File: ansible/roles/master/tasks/main.yml [hunk 1]
```diff
@@ -12,3 +12,12 @@
...
   when: squad_scheduler_service.changed or install_squad.changed or settings.changed or environmentfile.changed or linaro_ldap_backends.changed or install_squad_linaro_plugins.changed
   changed_when: False
+
+# It seems that journald/journalctl won't manage systemd-journal-remote's files.
+# Use a bigger hammer to keep size under control.
+- name: Remove old log files from /var/log/journal
+  cron:
+    name: Remove old log files from /var/log/journal
+    special_time: daily
+    job: /usr/bin/find /var/log/journal/ -type f -user systemd-journal-remote -mtime +14 -delete
```
Comment (terceiro) [inline ansible/roles/master/tasks/main.yml:22]: you probably want `journalctl --vacuum-time=14d` instead.

Comment (terceiro) [reply ansible/roles/master/tasks/main.yml:22]: just now noted that this is already merged, will do the change myself in that case

Comment (terceiro) [reply ansible/roles/master/tasks/main.yml:22]: nevermind, I just noticed the comment above about remote journal files not being removed :-(

**Pull request merged**